db/migrate/20171005115151_add_description_to_restaurant.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToRestaurant < ActiveRecord::Migration[5.1]
+  def change
+    add_column :restaurants, :description, :text
```

Comment:
Again, this story is NOT about restaurants. Please stick to the plan. I know I'm being a pain in the ass but this is a learning experience. In a real project this would cause a mess and be treated as a blocker. Stay in scope.

Reply:
You are only doing what we want you to do, being a pain in our arse. we only learn when we know what we did wrong.